AuRoRA/src/scs/scs/cnc.py

```
@@ -50,188 +350,819 @@ def load_chat_history(self):
+            if image_data.startswith('data:image'):
+                parts = image_data.split(',', 1)
+                if len(parts) != 2:
+                    raise ValueError("Malformed data URL")
+                image_base64 = parts[1]
+            else:
+                image_base64 = image_data
+            
+            # Validate base64
+            try:
+                base64.b64decode(image_base64, validate=True)
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)